catalog/views.py

```diff
@@ -1,3 +1,27 @@
 from django.shortcuts import render
+from .models import Book, Author, BookInstance, Genre
 
 # Create your views here.
+
+def index(request):
+    """View function for home page of site."""
+
+    # Generate counts of some of the main objects
+    num_books = Book.objects.all().count()
+    num_instances = BookInstance.objects.all().count()
```

Comment:
không cần dùng .all()
```suggestion
    num_instances = BookInstance.objects.count()
```